modules/Microsoft.Consumption/budgets/.test/default/deploy.test.bicep

```
@@ -0,0 +1,29 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. A short identifier for the kind of deployment .Should be kept short to not run into resource-name length-constraints')
```

Minor typo in this description. Applies to all PRs. Should probably be:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

---

Updated via automation :)